script.js

```diff
@@ -1,4 +1,294 @@
+    Please click enter to view the game outcome.`;
+
+    player2Score = `${player2dice2Arr}${player2dice1Arr}`; // Player 2 score
+
+    console.log(`player 2 score, ${player2Score}`);
+    console.log(`user Chosen Dice Order, ${userChosenDiceOrder}`);
+    console.log(`Dice 2 as first number results, ${myOutputValue}`);
+
+    currentGameMode = "gameOutcome"; // Change to game outcome mode
+    console.log(
+      `Both players turn has ended, moving on to game outcome. Current mode : ${currentGameMode}`
+    );
+    return myOutputValue;
+  } else if (
+    currentGameMode == "player2Choice" &&
+    (userChosenDiceOrder !== 2 || userChosenDiceOrder !== 1)
+  ) {
+    myOutputValue = `You have enter an invalid input for dice order. Pleaser enter either a 1 or 2 to choose your dice order. Thank you! <br><br>
+      Your rolled dice result: <br><br>
+    dice 1 : ${player2dice1Arr} <br><br>
+    dice 2 : ${player2dice2Arr} <br><br>
+    Please choose the order of desired dice result. <br>
+    Type 1 and click enter if you wish for dice 1 to be the first number <br>
+    Type 2 and click enter if you wish for dice 2 to be the first number
+      `;
+
+    console.log(
+      `Invalid input, current mode should still be player choice : ${currentGameMode}`
+    );
+    return myOutputValue;
+  }
+  return myOutputValue;
+};
+
+var gameOutcome = function (player1, player2) {
+  myOutputValue = "";
+  if (player1Score > player2Score) {
+    myOutputValue = `Congratz ${player1} for winning!!! <br><br>
+    ${player1} BEATS ${player2} with a score of ${player1Score} against ${player2Score} <br><br>
+    Click enter to start another new game`;
+    return myOutputValue;
+  } else if (player2Score > player1Score) {
+    myOutputValue = `Congratz ${player2} for winning!!! <br><br>
+     ${player2} BEATS ${player1} with a score of ${player2Score} against ${player1Score}<br><br>
+    Click enter to start another new game`;
+    return myOutputValue;
+  }
+};
+
+//Beat That Game Function
+var beatThatGame = function () {};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "";
+
+  if (currentGameMode == "welcomeToBeatThat") {
+    //Opening message + Instructions
+    myOutputValue = `Welcome to BEAT THAT!<br><br>
+     BEFORE starting the game, please ENTER desired player 1 In Game Name (IGN) and click ENTER, followed by player 2. <br><br>
+      Game Rules: <br>
+      1. This is a 2 person, player vs player game. <br>
+2. The game will roll 2 dice and show the dice rolls result. For example, dice 1 rolled 3 and dice 2 rolled 4. <br>
+3. The player that rolled the dice will pick the order of the dice.(If they wanted the number 43, they would specify that the 2nd dice goes first.) <br>
+4. After both players have rolled and chosen their dice order, the player with the higher combined number wins.  `;
+
+    currentGameMode = "enterPlayer1";
+
+    return myOutputValue;
+  }
```

Comment:
this is ok, but try seeing if you can output the message into the output box immediately on page load by editing the html so that the user sees the message straight away when he loads the game. Currently when I load your game, I am greeted with an empty text input and output box, and it is not immediately clear what I have to do. Always think about user experience when you are designing something!

This will involve editing the html which I realise you attempted a bit in this project by changing the header and button text but do try playing around with HTML/CSS more for blackjack, even if it is simple things like changing the background color of the page or the container box, or changing the fonts that are being used.